lib/dry/types/hash/schema.rb

```diff
@@ -149,8 +149,6 @@ def resolve(hash)
           end
         end
       end
-
-      private_constant(*constants(false))
```

Comment:
You can replace the splat with the actual list of constant names instead of removing it entirely